py/uncategorized_behavior_report.py

```diff
@@ -0,0 +1,72 @@
+'''
+    Filename: uncategorized_behavior_report.py
+    Author: Brett Coker
+    Python Version: 3.6.1
+
+    Generates an uncategorized beahvior report for IBM - Network and saves it
+    as a CSV file named uncategorized_behaviors.csv. The file is saved to the
+    directory where this script is run from.
+'''
+import requests
+import sys
+import csv
+from getpass import getpass
+
+# Two URLs that must be defined for working with the Lotame API
+api_url = 'https://api.lotame.com/2/'
+auth_url = 'https://crowdcontrol.lotame.com/auth/v1/tickets'
+
+
+def main():
+    username = input('Username: ')
+    password = getpass()
+    payload = {'username': username, 'password': password}
+
+    # Get the ticket-granting ticket from the Lotame API. If we get a KeyError,
+    # we know that the credentials are invalid, so we handle this by exiting
+    try:
+        tgt = requests.post(auth_url, data=payload).headers['location']
+    except KeyError:
+        print('Error: Invalid username and/or password.')
+        sys.exit()
+
+    # Get the client ID to create the report for
+    client_id = input('Client ID: ')
+
+    # This call will grab the first 5000 results from the report. If there are
+    # greater than 5000 behaviors, consider either increasing the page_count
+    # or creating a loop that utilizes the page_num paramenter of the API call
+    endpoint = 'reports/behaviors/uncategorized?client_id=' + \
```

Comment:
I'm hoping to start deprecating this endpoint, so lets direct people to the newer one:

https://api.lotame.com/2/behaviors/search/elastic?client_id=25&owner=OWNER&active=ACTIVE_ONLY&categorized=UNCATEGORIZED_ONLY&ignored=NON_IGNORED_ONLY&categorizable=CATEGORIZABLE_ONLY&date_range=LAST_30_DAYS&page_count=50&page_num=1&sort_attr=id&sort_order=ASC&appendNetworkClients=false&clientAsGroup=true

?

(little bit of re-writing then for the different format but faster!)


@sromack or should we leave this example out until we can merge the /search and /search/elastic into a single /search again?